docs/installation.rst

```diff
@@ -33,18 +35,21 @@ pysat itself may be installed from a terminal command line via::
    pip install pysat
 
 There are a few packages that pysat depends on that will be installed as
-needed by the installer
+needed by the installer:
 
-     * dask
-     * netCDF4
-     * numpy
-     * pandas
-     * portalocker
-     * scipy
-     * toolz
-     * xarray
+#. dask
+#. netCDF4
+#. numpy
+#. pandas
```

Comment:
Packaging?

Reply:
packaging is not currently used in the core package.  Verified via "find all" in atom.